src/modules/bids-asks/helpers/select-order-book.js

```diff
@@ -95,7 +95,9 @@ const selectAggregatePricePoints = memoize((outcomeID, side, orders, orderCancel
       const obj = {
         isOfCurrentUser: shareCountPerPrice[price].isOfCurrentUser,
         shares: formatShares(shareCountPerPrice[price].shares),
-        price: formatEtherTokens(parseFloat(price))
+        price: formatEtherTokens(parseFloat(price)),
```

Comment:
Does `formatEtherTokens` require a JS number input?  Recommend keeping numbers as either strings or BigNumbers (if you need to arithmetic with them); avoid JS numbers and their associated rounding errors.